spec/controllers/spree/user_registrations_controller_spec.rb

```diff
@@ -9,6 +9,19 @@
       expect(response).to redirect_to spree.account_path
     end
 
+    context "with non default locale" do
+      before do
+        Spree::Store.default.update(default_locale: 'en', supported_locales: 'en,fr') if Spree.version.to_f >= 4.2
+      end
+
+      after { I18n.locale = :en }
```

Comment:
Curious why you are setting this here?

Reply:
Becouse otherwise tests that are later in the file will have fr locale.

Reply:
Ah, usually rspec "cleans up" after each round of tests so that we have a fresh slate, but perhaps there are certain values that it does not reset. Ideally you would not have to manually reset like this.

I would look into seeing if you can stub something within `Spree::Store` regarding the locale so that you dont need to perform this "after" step.

Reply:
Actually maybe that would not be possible, looks like the locale is getting set via the `post :create` action. Cool, then we can leave this in since it may be necessary for tests to pass consistently.